src/components/FileManager/FilesSection.tsx

```diff
@@ -49,19 +48,13 @@ export const FilesSection: React.FC<{
     )
   }
   const isSupplementFilesTab = fileSection === FileSectionType.Supplements
-
   const isOtherFileTab = fileSection === FileSectionType.OtherFile
-  const handleUploadOtherFile = () => {
-    dispatch(actions.HANDLE_UPLOAD_ACTION())
-    state.uploadedFile &&
-      state.selectDesignation !== undefined &&
-      handleUpload(
-        state.uploadedFile,
-        getDesignationName(state.selectDesignation)
-      )
-  }
   const can = useContext(PermissionsContext)
-
+  const EMPTY_DESIGNATION = ''
+  useEffect(() => {
+    dispatch(actions.HANDLE_UPLOAD_ACTION())
+    state.uploadedFile && handleUpload(state.uploadedFile, EMPTY_DESIGNATION)
```

Comment:
Can't we remove the designation completely instead of sending an empty string?

Reply:
Done...